### ISSUE TITLE
BUG: Media library stops working after enabling plugin on multisite

### ISSUE THREAD

**Issue by pacifists** [baselbers/woocommerce-pdf-invoices#225]
Hello,

I've noticed that my pictures turned blank after enabling the plugin. Because the plugin changes the upload url.

I've found out that this method is the issue. What happens is that it makes file url into:
`/wp-content/uploads/sites/7/sites/7/....` instead of the normal url. So everytime after activating the plugin all pictures stop working.

```php
// file: woocommerce-pdf-invoices.php
// lines 64-72
public function setup_multisite_upload_dir( $upload_dir ) {
	// $upload_dir['basedir'] = $upload_dir['basedir'] . '/sites/' . get_current_blog_id();
	$upload_dir['path']    = $upload_dir['basedir'] . $upload_dir['subdir'];
	//$upload_dir['baseurl'] = $upload_dir['baseurl'] . '/sites/' . get_current_blog_id();
	$upload_dir['url']     = $upload_dir['baseurl'] . $upload_dir['subdir'];

	return $upload_dir;
}
```

In order to fix it. I had to comment out these lines as you can see.

**Comment by ablears** [comment]
Thanks, this fixes #221 too.

**Comment by baselbers** [comment]
@pacifists Thanks!
88b054b278710009eedc3c24a6a743da524bd18e
